### PR TITLE
[Cleanup] Update all records - Invoice design

### PR DESCRIPTION
**Pull request by Civolilah**
Here is the screenshot of the Invoice Design page after adding `Update all records` functionality:

Screenshot 2023-02-08 at 20 55 41

@beganovich @turbo124 Let me know your thoughts.

### CODE REVIEW

File: src/pages/settings/invoice-design/components/GeneralSettings.tsx [hunk 1]
```diff
@@ -797,63 +861,119 @@ export function GeneralSettings() {
...
+            {designs &&
+              designs.data.data.map((design: Design) => (
+                <option key={design.id} value={design.id}>
+                  {design.name}
+                </option>
+              ))}
+          </SelectField>
+
+          {isChangedDesign('credit') && (
+            <div className="flex space-x-10 pl-5">
```
Comment (beganovich) [inline src/pages/settings/invoice-design/components/GeneralSettings.tsx:937]: Padding.

File: src/pages/settings/invoice-design/components/GeneralSettings.tsx [hunk 2]
```diff
@@ -797,63 +861,119 @@ export function GeneralSettings() {
...
+              <span>{t('update_all_records')}</span>
+              <Toggle
+                checked={isUpdateAllRecordsChecked('invoice')}
+                onValueChange={(value) =>
+                  handleChangeUpdateAllRecords('invoice', value)
+                }
+              />
+            </div>
+          )}
+        </div>
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/settings/invoice-design/components/GeneralSettings.tsx]: Thanks for the tip, I'll keep this in mind for following features. Other than that, there's no particular reason to setting padding on the left. I just wanted to make a better UX/indication that this field is associated with one specific dropdown. It has now been removed and aligned with the dropdown menu.

File: src/pages/settings/invoice-design/components/GeneralSettings.tsx [hunk 3]
```diff
@@ -797,63 +861,119 @@ export function GeneralSettings() {
...
+            {designs &&
+              designs.data.data.map((design: Design) => (
+                <option key={design.id} value={design.id}>
+                  {design.name}
+                </option>
+              ))}
+          </SelectField>
+
+          {isChangedDesign('quote') && (
+            <div className="flex space-x-10 pl-5">
```
[the remark this answers is not in the document]
Comment (Civolilah) [reply src/pages/settings/invoice-design/components/GeneralSettings.tsx]: removed

File: src/pages/settings/invoice-design/components/GeneralSettings.tsx (hunk 1 above)
Comment (Civolilah) [reply src/pages/settings/invoice-design/components/GeneralSettings.tsx]: removed

**Review by beganovich — APPROVED** [review]
Looks good, thanks :+1:

**Pull request merged**